elasticsearch-model/test/integration/mongoid_import_test.rb

```diff
@@ -0,0 +1,155 @@
+require 'test_helper'
+
+begin
```

Comment:
The Mongo test setup has been recently extracted to test_helper.rb, I think here it will break for you.